backend/api.py

```
@@ -1907,6 +1907,1047 @@ def process_reaper():
+                    print("⚠️  Failed to initialize mechanisms")
+            except Exception as init_error:
+                print(f"⚠️  Error initializing mechanisms: {init_error}")
+                import traceback
+                traceback.print_exc()
+        
+        return jsonify({
+            'success': True,
+            'mechanisms': mechanisms,
+            'disclaimer': 'Research tool only - not for medical diagnosis or treatment'
+        })
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)